### PR TITLE
Ft/navbar

### LINKED ISSUE #3: Navbar
- Add nav links
- Add nav icons
- Implement active link
- Add logo 
- Make it responsive across all devices.

### PR DESCRIPTION
**Pull request by AdelineA**

### CODE REVIEW

File: src/Components/Navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <nav className="font-poppins py-4 relative">
+      <div className="hidden md:flex justify-between items-center">
+        <Logo />
+        <NavbarLinks />
+        <NavbarButtons />
```
Comment (kareraolivier) [inline src/Components/Navbar/index.tsx]: this can be navbarIcons

File: src/Components/Navbar/NavbarButtons/Search.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <div className="bg-gray-200 p-2 rounded-lg xl:flex gap-4 hidden">
+      <input
+        type="text"
+        placeholder="What are you looking for?"
+        className="bg-inherit outline-none text-[12px] text-black"
+      />
+      <SearchIcon />
+    </div>
+  );
+};
```
Comment (kareraolivier) [inline src/Components/Navbar/NavbarButtons/Search.tsx]: search should not be in navbar buttons

File: src/Components/Navbar/NavbarLinks.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+];
+const NavbarLinks = () => {
+  return (
+    <div>
+      <ul className="flex gap-5 font-normal text-base justify-center text-center">
+        {navLinks.map((link) => (
+          <li key={link.title} className="hover:underline">
+            <Link to={link.target}>{link.title}</Link>
+          </li>
+        ))}
```
Comment (kareraolivier) [inline src/Components/Navbar/NavbarLinks.tsx]: we need to know active links

**Review by kareraolivier — CHANGES_REQUESTED** [review]
## Few notes
- Create all pages to navigate to
- Create 404 page
- Pages and Routes should not be in components
-

File: src/Components/Navbar/index.tsx (hunk 1 above)
Comment (AdelineA) [reply src/Components/Navbar/index.tsx]: Fixed

File: src/Components/Navbar/NavbarButtons/Search.tsx (hunk 2 above)
Comment (AdelineA) [reply src/Components/Navbar/NavbarButtons/Search.tsx]: done

File: src/Components/Navbar/NavbarLinks.tsx (hunk 3 above)
Comment (AdelineA) [reply src/Components/Navbar/NavbarLinks.tsx]: done

File: src/Routes/Routes.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Layout />,
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        path: "",
+        element: <Homepage />,
```
Comment (kareraolivier) [inline src/Routes/Routes.tsx]: path can be index here

File: src/Routes/Routes.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+import About from "../pages/About";
+import SignUp from "../pages/SignUp";
+import ErrorPage from "../pages/404";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Layout />,
+    errorElement: <ErrorPage />,
+    children: [
```
Comment (kareraolivier) [inline src/Routes/Routes.tsx]: there is no navbar on error page

File: src/Components/Navbar/Search.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import SearchIcon from "../../assets/Icons/SearchIcon";
+
+const Search = () => {
+  return (
+    <div className="bg-gray-200 p-2 rounded-lg xl:flex gap-4 hidden">
```
Comment (kareraolivier) [inline src/Components/Navbar/Search.tsx]: increase the width of this input
also on mobile we have to see search

**Review by kareraolivier — CHANGES_REQUESTED** [review]
This is great, I left some minor review

**Pull request merged**